dali/kernels/signal/dct/dct_gpu.cu

```diff
@@ -29,11 +29,49 @@ namespace kernels {
 namespace signal {
 namespace dct {
 
+struct Lifter {
+  DALI_HOST_DEV
+  explicit Lifter(float coeff): coeff_(coeff) {}
+
+  DALI_HOST_DEV
+  float operator()(float val) { return val * coeff_; }
+
+  const float coeff_;
```

Comment:
Either:
```suggestion
  const float coeff;
```
or
```suggestion
 private:
  const float coeff_;
```

Reply:
I would go for the second as you don't want to access `coeff_` directly anyway.

Reply:
I've remove LifterTabble